### DISCUSSION TITLE
Error parsing JSON response on Intellij

### DISCUSSION THREAD

**Discussion by jbmsousa** [quarkusio/quarkus#33327] [Q&A]
I aim trying to create a quarkus project on intellij but a have this error:Initialization failed for 'https://code.quarkus.io'
Please check URL, network and proxy settings.

Error message:
Error parsing JSON response

This url is accessible from my netwok, i have no proxy.
Please help me.
Best regards,
Sousa

**Comment by bulldog98** [comment]
I was unable to reproduce that. Could you provide more details where that failed also that creation page is maintained by the Intellij Team, so this should be reported there.
